includes/admin/onboarding/steps/step-tools.php

```diff
@@ -0,0 +1,200 @@
+					<div class="edd-onboarding__plugins-details">
+						<h3><?php echo esc_html( $plugin['name'] ); ?></h3>
+						<p><?php echo esc_html( $plugin['description'] ); ?></p>
+					</div>
+					<div class="edd-onboarding__plugins-control">
+
+					<label class="checkbox-control checkbox-control--checkbox">
+						<input class="edd-onboarding__plugin-install" data-plugin-name="<?php echo esc_attr( $plugin['plugin_name'] ); ?>" data-action="<?php echo esc_attr( $plugin['action'] ); ?>" data-plugin-file="<?php echo esc_attr( $plugin['plugin_file'] ); ?>" value="<?php echo esc_attr( $plugin['plugin_url'] ); ?>" type="checkbox"<?php echo $checked.$disabled;?>/>
+						<div class="checkbox-control__indicator"></div>
+					</label>
+
+					</div>
+				</div>
+				<?php
+			endforeach;
+			?>
+		</div>
+
+		<div class="edd-onboarding__get-suggestions-section">
+			<h3>Get helpful suggestions from Easy Digital Downloads on how to supercharge your EDD powered store, so you can improve conversions and earn more money.</h3>
+
+			<div class="edd-onboarding__get-suggestions-section_label">
+				<label for="get-suggestion-email">
+					<?php echo esc_html( __( 'Your Email Address:', 'easy-digital-downloads' ) ); ?>
+				</label>
+			</div>
+
+			<div class="edd-onboarding__get-suggestions-section_input">
+				<input type="email" name="edd_telemetry_email" value="<?php echo esc_attr( $telemetry_email ); ?>" id="edd-onboarding__telemery-email">
+			</div>
+
+			<label class="edd-toggle">
+				<input type="checkbox" id="edd-onboarding__telemery-toggle" name="telemetry" value="1" checked> <?php echo esc_html( __( 'Help make EDD better for everyone', 'easy-digital-downloads' ) ); ?>
+			</label>
+			<span alt="f223" class="edd-help-tip dashicons dashicons-editor-help" title="<?php echo esc_attr( __( 'Explanation for the telemetry.', 'easy-digital-downloads' ) ); ?>"></span></input>
+		</div>
+
+		<div class="edd-onboarding__selected-plugins">
+			<p><?php echo esc_html( __( 'The following plugins will be installed:', 'easy-digital-downloads' ) ); ?> <span class="edd-onboarding__selected-plugins-text"></span></p>
+		</div>
+	</div>
+	<div class="edd-onboarding__install-failed" style="display: none;">
+		<h3><?php echo esc_html( __( 'Some features were not able to be installed!', 'easy-digital-downloads' ) ); ?></h3>
+		<p>
+			<?php
+				/* Translators: list of plugins that were not able to be installed or activated */
+				wp_kses(
+					printf( __( "Don't worry, everything will still work without them! You can install %s later by going to the Plugins > Add New section of your admin", 'easy-digital-downloads' ), '<span class="edd-onboarding__failed-plugins-text"></span>' ),
+					array( 'span' )
+				);
+			?>
+		</p>
+		<?php echo esc_html( __( 'Your Email Address:', 'easy-digital-downloads' ) ); ?>
+		<a href="#" class="button button-primary edd-onboarding__button-skip-step"><?php echo esc_html( __( 'Continue', 'easy-digital-downloads' ) ); ?></a>
```

Comment:
This should be a button, not a link. Sorry, I thought those were all in a helper function.

Reply:
Actually should this and the line above be removed completely? I think they're left over from something else.